src/Core/Entities/PlayerEntity.m

```diff
@@ -7911,21 +7911,24 @@ - (NSString *) primedEquipmentName:(NSInteger)offset
 }
 
 
-- (BOOL) setPrimedEquipment:(NSString *)eqKey
+- (BOOL) setPrimedEquipment:(NSString *)eqKey showMessage:(BOOL)showMsg
 {
 	NSUInteger c = [eqScripts count];
 	NSUInteger current = primedEquipment;
 	primedEquipment = [self eqScriptIndexForKey:eqKey];	// if key not found primedEquipment is set to primed-none
 	BOOL result = YES;
-	if (primedEquipment == c)
+	if (primedEquipment == c && ![eqKey isEqualToString:@""])
 	{
 		primedEquipment = current;
 		result = NO;
 	}
 	else 
 	{
-		NSString *equipmentName = [[OOEquipmentType equipmentTypeWithIdentifier:[[eqScripts oo_arrayAtIndex:primedEquipment] oo_stringAtIndex:0]] name];
-		[UNIVERSE addMessage:OOExpandKey(@"equipment-primed", equipmentName) forCount:2.0];
+		if (showMsg == YES)
```

Comment:
Suggest modification to `if (primedEquipment != current && showMsg)` to avoid occurence of messages about priming equipment that is already primed at that time.